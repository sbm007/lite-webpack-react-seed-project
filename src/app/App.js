import React from 'react';
import { hot } from 'react-hot-loader';

import './App.scss';

export const App = hot(module)(() => (
  <div className="App">
    <p>Hello world!</p>
  </div>
));
