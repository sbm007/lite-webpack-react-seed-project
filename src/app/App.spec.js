import React from 'react';
import { shallow } from 'enzyme';

import { App } from '~/App';

describe('App', () => {

  test('render the <App> component', () => {
    const wrapper = shallow(<App/>);
    expect(wrapper).toMatchSnapshot();
  });

});
