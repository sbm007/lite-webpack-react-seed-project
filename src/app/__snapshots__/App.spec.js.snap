// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render the <App> component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        Hello world!
      </p>,
      "className": "App",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Hello world!",
      },
      "ref": null,
      "rendered": "Hello world!",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          Hello world!
        </p>,
        "className": "App",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hello world!",
        },
        "ref": null,
        "rendered": "Hello world!",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
